curriculum/challenges/english/14-responsive-web-design-22/learn-css-grid-by-building-a-magazine/6143a778bffc206ac6b1dbe3.md

```diff
@@ -10,12 +10,7 @@ dashedName: step-17
 Create one final `p` element at the end of your `.text` element and give it the following text:
 
 ```markup
-Instead of a series of coding challenges, people will be in their code
-editor passing one test after another, quickly building up a project.
-People will get into a real flow state, similar to what they
-experience when they build the required projects at the end of each
-certification. They'll get that sense of forward progress right from
-the beginning. And freeCodeCamp will be a much smoother experience.
+Instead of a series of coding challenges, people will be in their code editor passing one test after another, quickly building up a project. People will get into a real flow state, similar to what they experience when they build the required projects at the end of each certification. They'll get that sense of forward progress right from the beginning. And freeCodeCamp will be a much smoother experience.
```

Comment:
This doesn't really solve the problem because there is still a horizontal scroll bar. In fact, I'd argue it makes it worse because now it is one very long line. At least before, the lines were broken up a little and the horizontal scroll wasn't as long.

The real solution here is to do away with preserving white-space and honoring line breaks for this type of text in which strict formatting is not required. Then the text can flow normally in its container. I mentioned this in issue [#50834](https://github.com/freeCodeCamp/freeCodeCamp/issues/50834).

Reply:
This fixes the issue of ugly formatting but we should just add the ability to have a horizontal scrollbar in. If you want to utilize white space as you said, we need to go through every step and do that so that things are consistant. To keep it simpler, we add the ability to use a scroll bar and keep this step consistent and on one line just like the rest of the steps.